src/main/kotlin/org/wycliffeassociates/otter/jvm/app/theme/AppStyles.kt

```diff
@@ -20,6 +20,7 @@ class AppStyles : Stylesheet() {
         val appBackground by cssclass()
         val progressDialog by cssclass()
         val appToggleButton by cssclass()
+        val workingArea by cssclass()
```

Comment:
This quite possibly doesn't belong here- perhaps in MainScreenStyles instead?

Reply:
yeah, probably should go in MainScreenStyles

Reply:
done